src/components/Search/index.tsx

```diff
@@ -224,6 +224,14 @@ function Search({queryJSON, currentSearchResults, lastNonEmptySearchResults, onS
         handleSearch({queryJSON, offset});
     }, [handleSearch, isOffline, offset, queryJSON]);
 
+    useEffect(() => {
+        if (isOffline) {
+            return;
+        }
+
+        openSearch();
+    }, [isOffline]);
+
```

Comment:
```suggestion
    useEffect(() => {
        openSearch();
    }, []);

```

It should persist once called, to avoid calling it multiple times when going online or offline. Also, it will be called every time you switch tabs, which doesn't seem ideal. What do you think?